scripts/setup.sh

```diff
@@ -256,30 +257,5 @@ export PYTHON_CMD
 
 # List all node modules that are currently installed. The "npm list" command is
 # slow, so we precompute this here and refer to it as needed.
-echo "Generating list of installed node modules..."
-NPM_INSTALLED_MODULES="$($NPM_CMD list)"
-export NPM_INSTALLED_MODULES
 echo "Generation completed."
```

Comment:
Probably not relevant since you removed the "Generating list of ..." line

Reply:
Removed.